CDCgov/cdc-open-viz
# Enhancement [DEV-9861] Improve map territory labels

## [DEV-9861](https://websupport-cdc.msappproxy.net/browse/DEV-9861)

The OPHDST team would like to make some improvements to territory labels shown below maps. See the attached screenshot for direction.

## Testing Steps
1. Create or view map with territories
2. Observe new layout

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)

### Before
<img width="511" alt="388241181-86917acb-e53e-4f89-a7dd-78c23cd73289" src="https://github.com/user-attachments/assets/5097d6ba-099a-43f0-9373-ff0deeb0b399">

### After
<img width="329" alt="Screenshot 2024-11-21 at 3 23 19 PM" src="https://github.com/user-attachments/assets/bba09dae-2fe9-4775-9ec6-495a01e83005">
